docs/storage/s3/parquet.md

```diff
@@ -0,0 +1,38 @@
+# Apache Parquet
+
+There is idea to use Apache Parquet for storing group relations between Runtime and the Application.
```

Comment:
I just missing some data structure in S3 itself, something similar you did for s3 documentation. About Parquet file and where specs are stored.

Reply:
Done